### PR TITLE
scib & dependencies

### PR DESCRIPTION
**Pull request by deniskristak**
some dependencies of `scib-pipeline` https://github.com/vscentrum/vsc-software-stack/issues/33

adding easyconfigs: 
`Deprecated-1.2.13-foss-2022a.eb`
~`numba-0.56.4-foss-2022a.eb`~
`pydot-1.4.2-GCCcore-11.3.0.eb`
~`scanpy-1.9.1-foss-2022a.eb`~
`scib-1.1.1-foss-2022a.eb`
`scikit-misc-0.1.4-foss-2022a.eb`
~`umap-learn-0.5.3-foss-2022a.eb`~

requires:
* https://github.com/easybuilders/easybuild-easyconfigs/pull/17143

### CODE REVIEW

File: easybuild/easyconfigs/s/scanpy/scanpy-1.9.1-foss-2022a.eb [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    ('anndata', '0.8.0', {
+        'checksums': ['94d2cc6f76c0317c0ac28564e3092b313b7ad19c737d66701961f3e620b9066e'],
+    }),
+    ('pynndescent', '0.5.8', {
+        'checksums': ['a7c552569bf604a101fd54bba1d27c12389e065945dee3a6777a518c63a46f2b'],
+    }),
+    ('umap-learn', '0.5.3', {
+        'modulename': 'umap',
+        'checksums': ['dbd57cb181c2b66d238acb5635697526bf24c798082daed0cf9b87f6a3a6c0c7'],
+    }),
```
Comment (jfgrimm) [inline easybuild/easyconfigs/s/scanpy/scanpy-1.9.1-foss-2022a.eb:49]: depend on the `umap-learn` easyconfig instead

File: easybuild/easyconfigs/u/umap-learn/umap-learn-0.5.3-foss-2022a.eb [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    ('scikit-learn', '1.1.2'),
+    ('numba', '0.56.4'),
+    ('LLVM', '14.0.3'),
+    ('tqdm', '4.64.0'),
+]
+
+use_pip = True
+
+exts_list = [
+    ('pynndescent', '0.5.8'),
```
Comment (jfgrimm) [inline easybuild/easyconfigs/u/umap-learn/umap-learn-0.5.3-foss-2022a.eb:27]: missing checksum

File: easybuild/easyconfigs/s/scanpy/scanpy-1.9.1-foss-2022a.eb (hunk 1 above)
Comment (deniskristak) [reply easybuild/easyconfigs/s/scanpy/scanpy-1.9.1-foss-2022a.eb]: solved

File: easybuild/easyconfigs/u/umap-learn/umap-learn-0.5.3-foss-2022a.eb (hunk 2 above)
Comment (deniskristak) [reply easybuild/easyconfigs/u/umap-learn/umap-learn-0.5.3-foss-2022a.eb]: added

**Comment by deniskristak** [comment]
https://github.com/easybuilders/easybuild-easyconfigs/pull/17143
this should go first

**Comment by verdurin** [comment]
@deniskristak I've merged #17143 - could you look at the reported conflicts, please?

**Comment by deniskristak** [comment]
@verdurin I think we just need to run the test again (test report is +- 1.5 hours older than when you merged it)

**Comment by deniskristak** [comment]
@verdurin done

**Comment by verdurin** [comment]
@deniskristak still one conflict reported.

**Comment by deniskristak** [comment]
@verdurin done, but based on [this](https://github.com/vscentrum/vsc-software-stack/issues/74), it might be obsolete at this point? or at least `scanpy` ?
@boegel what do you think?

File: easybuild/easyconfigs/s/scanpy/scanpy-1.9.1-foss-2022a.eb [hunk 3]
```diff
@@ -40,6 +41,9 @@ exts_list = [
     ('anndata', '0.8.0', {
         'checksums': ['94d2cc6f76c0317c0ac28564e3092b313b7ad19c737d66701961f3e620b9066e'],
     }),
+    ('pynndescent', '0.5.8', {
+        'checksums': ['a7c552569bf604a101fd54bba1d27c12389e065945dee3a6777a518c63a46f2b'],
+    }),
```
[the remark this answers is not in the document]
Comment (jfgrimm) [reply easybuild/easyconfigs/s/scanpy/scanpy-1.9.1-foss-2022a.eb:46]: i'd probably just remove `scanpy` changes from this PR

[the remark this answers is not in the document]
Comment (deniskristak) [reply easybuild/easyconfigs/s/scanpy/scanpy-1.9.1-foss-2022a.eb]: fair, removed :)

**Comment by jfgrimm** [comment]
Test report by @jfgrimm
**SUCCESS**
Build succeeded for 5 out of 5 (4 easyconfigs in total)
node087.pri.viking.alces.network - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6138 CPU @ 2.00GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/85e48b7d88b77fb8297bde1f9658d929 for a full test report.

**Comment by jfgrimm** [comment]
@boegelbot please test @ generoso

**Comment by jfgrimm** [comment]
@boegelbot please test @ jsc-zen2

**Review by jfgrimm — APPROVED** [review]
lgtm

**Comment by jfgrimm** [comment]
Going in, thanks @deniskristak!

**Pull request merged**